packages/cattle-pushprox/README.md

```diff
@@ -0,0 +1,52 @@
+# cattle-pushprox
+
+A Rancher chart based on [PushProx](https://github.com/prometheus-community/PushProx) that sets up a deployment of the PushProx proxy and several PushProx clients on a Kubernetes cluster.
+
+Installs [cattle-pushprox](https://github.com/rancher/dev-charts/tree/master/packages/cattle-pushprox) to create a DaemonSet of clients that can access their host's network and register with a PushProx proxy Deployment. A [Prometheus Operator](https://github.com/coreos/prometheus-operator) ServiceMonitor CR is also included that is configured to scrape the metrics from each of the clients through the proxy.
+
+The clients and proxy are created based on of a Rancher fork of the Prometheus Community [PushProx](https://github.com/prometheus-community/PushProx) project.
```

Comment:
```suggestion
The clients and proxy are created based on a Rancher fork of the Prometheus Community [PushProx](https://github.com/prometheus-community/PushProx) project.
```

Reply:
Good catch, will be updated in the latest commit